src/webgpu/api/validation/encoding/cmds/copyTextureToTexture.spec.ts

```diff
@@ -510,6 +512,51 @@ g.test('copy_within_same_texture')
     );
   });
 
+// TODO(jiawei.shao@intel.com): test 'stencil-only' is also valid for the texture format 'stencil8'
+g.test('copy_aspects')
+  .params(
+    params()
+      .combine(
+        poptions('formatAndValidAspects', [
+          { format: 'rgba8unorm', validAspects: ['all'] },
+          { format: 'depth24plus-stencil8', validAspects: ['all'] },
+          { format: 'depth32float', validAspects: ['all', 'depth-only'] },
+        ] as const)
+      )
```

Comment:
nit: use something like this:
```suggestion
      .combine([
        { format: 'rgba8unorm', _validAspects: ['all'] },
        { format: 'depth24plus-stencil8', _validAspects: ['all'] },
        { format: 'depth32float', _validAspects: ['all', 'depth-only'] },
      ] as const)
```
The underscore makes it "private" (not part of the test path).

Reply:
The change you made doesn't make _validAspects private, as there is still a parameter `formatAndValidAspects` with value `{'format':'rgba8unorm','_validAspects':['all']}`. You should be able to see this in the standalone runner, in the query strings of the cases.

Instead, I think you should do something like this which makes two different parameters, `format`, and `_validAspects` which is private.

Reply:
Actually, come to think of it, the information of which aspects are valid for which format should ultimately live in capability_info.ts. No need to do it now, it'll be a bit messy (have to do it for every format). Just leave a TODO here saying the info should be pulled from capability_info once it's added there.